lib/taurus/qt/qtgui/panel/taurusvalue.py

```diff
@@ -1170,8 +1174,7 @@ def createConfig(self, allowUnpickable=False):
                     configdict[key]['delegate'] = widget.createConfig()
             else:
                 self.info('createConfig: %s not saved because it is not Pickable (%s)' % (
-                    key, str(classID)))
-
+                        key, str(classID)))
```

Comment:
Not critical, but... why this change?  AFAIK, the correct indentation according to PEP8 is the original one  (not that this file is very PEP8-complyiant, but...)

Reply:
Fixed in 4cf0ef0